src/infrastructure/external_ack.rs

```diff
@@ -180,14 +184,6 @@ mod test {
         acks.ack(6);
         acks.ack(4);
         assert_eq!(acks.last_seq, 6);
-        assert_eq!(
```

Comment:
This might be removed, I have one concern to this which is that people will never know what `0b110010` means. The previous code was more cleaner.

Reply:
the bit shifts, really? what if i commented it, it seems clearer to me this way. but if you prefer i will reset it

Reply:
I am a bit unfamiliar with that too, however, a random number might be more unclear, What does it mean, at least if you get the general idea of bit-shifting it will be easier to understand.

Reply:
What about "0b01 | 0b10" ?